src/DocumentEditor.ts

```diff
@@ -101,6 +101,11 @@ export class DocumentEditor implements vscode.Disposable {
 
     private async udpateDocumentToNode(node: IDocumentNode, doc: vscode.TextDocument): Promise<void> {
         const updatedDoc: {} = await node.update(JSON.parse(doc.getText()));
+        const output = util.getOutputChannel();
+        const timestamp = Math.floor(Date.now());
+        const docLink = node.getDocLink();
+        output.appendLine(`${timestamp}: Updated document "${node.id}" to collection "${docLink}"`);
```

Comment:
I'm fine with you using docLink, but then `node.id` is redundant information. Just do:
```
output.appendLine(`${timestamp}: Updated document "${docLink}"`);
```